{"name": "test_Wijken_failure", "status": "passed", "description": "Failure API request for the Wijken", "steps": [{"name": "API Call `https://acc.api.data.amsterdam.nl/v1/gebieden/wijken/0363097000000C to validate whether the no record found with `failure code 404`", "status": "passed", "start": 1647264620962, "stop": 1647264621099}], "start": 1647264620962, "stop": 1647264621099, "uuid": "b0908489-911d-4a43-9a7c-1c938259d988", "historyId": "037430a115a2552756c28c3061e688ed", "testCaseId": "42c285f4cbf7c1d3064bb577bde9fffe", "fullName": "e2etest.test_datasetgebiedenwijken#test_Wijken_failure", "labels": [{"name": "parentSuite", "value": "e2etest"}, {"name": "suite", "value": "test_datasetgebiedenwijken"}, {"name": "host", "value": "Jayashrees-MacBook-Pro.local"}, {"name": "thread", "value": "84720-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2etest.test_datasetgebiedenwijken"}]}